src/configs/nuage/elasticsearch/index.js

```diff
@@ -116,13 +123,56 @@ const getRequestID = function (queryConfiguration, context) {
     if (Object.keys(parameters).length === 0)
         return queryConfiguration.id;
 
-    return queryConfiguration.id + "[" + JSON.stringify(parameters) + "]";
+    return `${queryConfiguration.vizID}-${queryConfiguration.id}[${JSON.stringify(parameters)}]`;
+}
+
+const addSorting = function (queryConfiguration, sort) {
+    let tmpQueryConfiguration = {...queryConfiguration};
```

Comment:
there is no need to create an extra variable here! Just modify the existing queryConfiguration to add sort info. Let's avoid creating extra object to give GC more work. Put some comments for people to understand that the passed configuration is going to be modified.